README.md

```diff
@@ -1,16 +1,61 @@
 # Overview
-This package can be added to your workspaces to provide job runners
-that can fulfill resources requested by plans.
+This package can be used to create a wake `runner` that will set environment
+variables for jobs from JSON values based on the resources they request. A
+default runner is also included with this package.
 
 # Resources
-Resources describe a tool that is requested by a job. They are strings
-that have the format `{vendor}/{tool}/{version}`. If there is no vendor
-then the format is `{tool}/{version}`.
+Resources describe a tool that is requested by a job. They are strings.
+This package expects resources to have the format `{vendor}/{tool}/{version}`.
+If there is no vendor then the format is `{tool}/{version}`.
 
-# Runners
-This package provides a runner that sets the `PATH` of wake
-jobs to the value of the `WAKE_PATH` environment variable if it is set
-by the user.
+# Usage
+To use this you need to create a JSON file that maps resources to a JSON object
+representing the environment variables you want to set when that resource is
+requested.
```

Comment:
I know it's shown in the example, but I think it'd be good to explicitly mention that the outer portion of mapping is a series of nested objects, where the keys form a path corresponding to the `vendor/tool/version` format above.